src/scenes/home/landing/membership/membership.css

```diff
@@ -27,6 +27,10 @@
   padding-bottom: 20px;
 }
 
+.list li p {
+  font-size: 25px;
```

Comment:
Instead of hard-styling list paragraph elements, why not try one of the header elements, such as `<h6>`

Reply:
Good point! I went ahead and tried that instead, and I see that the styling actually retains consistency when doing this, and no need to style the paragraph element to look like the header.